cel/cos_tlv.go

```diff
@@ -79,3 +85,38 @@ func (t TLV) ParseToCosTlv() (CosTlv, error) {
 func (t TLV) IsCosTlv() bool {
 	return t.Type == CosEventType
 }
+
+// ValidateEnvVar takes in a environment name and its value, run some checks and return an error
+// if they are invalid.
+func ValidateEnvVar(name string, value string) error {
+	if !utf8.ValidString(name) {
+		return fmt.Errorf("malformed env name: [%s] (contains non-utf8 character)", name)
+	}
+	if !utf8.ValidString(value) {
+		return fmt.Errorf("malformed env value: [%s] (contains non-utf8 character)", value)
+	}
+	match, err := regexp.MatchString("^[a-zA-Z_][a-zA-Z0-9_]*$", name)
+	if err != nil {
+		// only if the pattern is invalid
+		panic(err)
+	}
+	if !match {
+		return fmt.Errorf("malformed env name: [%s] (doesn't match the regex ^[a-zA-Z_][a-zA-Z0-9_]*$)", name)
```

Comment:
should we have this be more human reaadable? something like "env name must start with an alpha character or _ , followed by a string of alphanumeric characters or _ "

Reply:
added the description